src/reference/glossary.rst

```diff
@@ -0,0 +1,75 @@
+========
+Glossary
+========
+
+.. glossary::
+
+   Augur
+
+      A command-line application used for phylogenetic analysis. :doc:`Documentation<augur:index>`
+
+   Auspice
+
+      A web application used for phylogenetic visualization and analysis. :doc:`Documentation<auspice:index>`
+
+   workflow
+      also *pathogen workflow*, *pathogen analysis*, *Nextstrain workflow*
+
+      A reproducible process comprised of one or more :term:`builds<build>` producing :term:`datasets<dataset>`, which can be visualized by :term:`Auspice`. Implementation varies per workflow, but generally they are run by workflow managers such as Snakemake.
+
+      Our core workflows can be divided into two types:
```

Comment:
Let's also define "core" 😄 https://docs.nextstrain.org/en/latest/guides/share/index.html is the relevant page, although the word "core" doesn't appear there at all!

Reply:
Added "core workflow": e3099ca619da192d18f891241e901d7f748ce0ed

We can update docs to use the new term in another PR.